packages/cozy-harvest-lib/src/components/hoc/withLocales.js

```diff
@@ -0,0 +1,8 @@
+import withLocales from 'cozy-ui/transpiled/react/I18n/withLocales'
+
+const locales = {
+  en: require(`../../locales/en.json`),
+  fr: require(`../../locales/fr.json`)
+}
+
+export default withLocales(locales)
```

Comment:
do you still need this file ? When you do `import withLocales from './hoc/withLocales'` is it the jsx or the js ?

Reply:
No we don't need it anymore, it was supposed to be removed in the last commit, please ignore it :)